docs/src/content/docs/challenges/angular/10-utility-wrapper-pipe.md

```diff
@@ -14,13 +14,13 @@ sidebar:
 
```

Comment:
If you want you can add your github handle in all files you fixed. if you want to.

thanks a lot 🙏

Reply:
Done in a new commit.

I also rebased.